selectors/views/custom_status.ts

```diff
@@ -15,7 +15,7 @@ export function makeGetCustomStatus(): (state: GlobalState, userID?: string) =>
     return createSelector(
         (state: GlobalState, userID?: string) => (userID ? getUser(state, userID) : getCurrentUser(state)),
         (user) => {
-            const userProps = user.props || {};
+            const userProps = user && user.props ? user.props : {};
```

Comment:
Would it make sense to simplify to:
```suggestion
            const userProps = user?.props || {};
```